source/core/parameters.cc

```diff
@@ -2252,164 +2264,191 @@ namespace Parameters
         "number of particles",
         "1",
         Patterns::Integer(),
-        "Number of particles represented by IB max number of particles = 10 ");
-      prm.declare_entry(
-        "initial refinement",
-        "0",
-        Patterns::Integer(),
-        "number of refinement around the particles before the start of the simulation ");
-      prm.declare_entry(
-        "stencil order",
-        "2",
-        Patterns::Integer(),
-        "Order of the stencil used for extrapolation to the boundary.");
+        "Number of particles reprensented by IB max number of particles = 10 ");
```

Comment:
Maybe add a comment on how more particles are allowed if load from file is used.